keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -482,6 +485,78 @@ are properly reloading tokens by:
   - upgrade test:
     test/e2e/upgrades/serviceaccount_admission_controller_migration.go
 
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+- **How can a rollout fail? Can it impact already running workloads?**
```

Comment:
1. Creation of CA config map can fail due to permission / quota / admission errors. Describe the metric of the `root_ca_cert_publisher` controller that would indicate issues here.
2. Newly issued tokens could fail to be recognized by skewed API servers not configured with the bound token signing key/issuer. Describe the metric in the (n-1) API servers that would indicate an increase in authentication errors.

Reply:
done.

Reply:
as I recall, I've had a problem where part of the configuration changed and clients (including kube-controller-manager) did not refresh their tokens.  That should either be addressed here or updates made to client-go to refresh on a sufficient 401s.

Reply:
could you elaborate more on this failure mode? iirc, kcm is usually use pre generated kubeconfig instead of incluster config.

Reply:
opened https://github.com/kubernetes/kubernetes/issues/98885 to track this